### PR TITLE
Iterable web device mode documentation

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Added docs on Iterable web device mode and updated the existing structure.

## Type of documentation

- [x] New documentation
- [x] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/destinations/streaming-destinations/iterable/iterable-device-mode.mdx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    }
+  ],
+})
+```
+
+The following table lists the mappings between the RudderStack event properties and the Iterable properties in case of the purchase events:
+
+| RudderStack property | Iterable property |
+| :--------------| :---------|
+| `properties.name` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `items[].name` |
```
[the remark this answers is not in the document]
Comment (utsabc) [reply docs/destinations/streaming-destinations/iterable/iterable-device-mode.mdx:89]: yes better to give an example where we will have array of products @ameypv-rudder can you take this example
https://www.rudderstack.com/docs/event-spec/ecommerce-events-spec/ordering/#order-completed

**Comment by utsabc** [comment]
@ameypv-rudder can we share this image as en example of web push would look for the users
Screenshot 2022-10-15 at 4 52 01 PM

File: docs/destinations/streaming-destinations/iterable/iterable-device-mode.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (ishachopra0401) [reply docs/destinations/streaming-destinations/iterable/iterable-device-mode.mdx:93]: Added, as suggested.

**Comment by utsabc** [comment]
> @ameypv-rudder can we share this image as en example of web push would look for the users Screenshot 2022-10-15 at 4 52 01 PM

@ishachopra0401 can we add this

**Review by utsabc — APPROVED** [review]

**Pull request merged**